src/index.js

```diff
@@ -154,6 +155,14 @@ window.wikiaJWPlayer = function (elementId, options, callback) {
 
 		wikiaJWPlayerAllowControllOnTouchDevices(playerInstance);
 
+		wikiaJWPlayerUserIntendedPlayControl(
+			//user coming from footer must've clicked the video thumbnail
+			window.location.search.indexOf('wikia-footer-wiki-rec') > -1,
```

Comment:
not sure if we should do it now, but how about extracting list of such queryparameters to some array? I assume there will be more places we would count as UIP

Reply:
That makes sense but we I will actually move this logic to app repository and implement it there :)